Source/RimConnection/Managers/AlertManager.cs

```diff
@@ -23,25 +23,54 @@ private static void EventNotification(string label, string description, LetterDe
                 Find.LetterStack.ReceiveLetter(new TaggedString(label), new TaggedString(description), letterDef, new LookTargets(newVector, currentMap));
             }
         }
-
+        
         public static void BadEventNotification(string description)
         {
             EventNotification("Twitch Event", description, badTwitchEventLetterDef, null );
         }
-
+        
         public static void BadEventNotification(string description, IntVec3 location)
         {
             EventNotification("Twitch Event", description, badTwitchEventLetterDef, location);
         }
 
+        // Not sure if duplicating the notification event is better or worse than making it call itself but this looked neater to me. -Alice
```

Comment:
This seems fine, I think the old version will probably be replaced entirely by this new one eventually